app/index.js

```diff
@@ -237,17 +242,18 @@ Generator.prototype.somethingsDifferent = function() {
 
 	if (this.conf.get('submodule') || this.conf.get('customDirs')) {
 
-		var me = this,
-			done = this.async();
+		var me = this;
+			//done = this.async();
```

Comment:
no, clean up needed. with the new fs, it seemed like this function didn't need to run asynchronously any more.